.github/workflows/validate.yml

```diff
@@ -31,3 +31,15 @@ jobs:
             git diff --stat
             exit 1
           fi
+    - name: Scan for accessibility issues
+      task: accessibility-insights.prod.task.accessibility-insights@3
+      condition: and(succeeded(), eq(variables['Build.Reason'], 'PullRequest'))
+      enabled: true
+      inputs:
+        baselineFile: "$(Build.SourcesDirectory)/baselines/vcpkg.github.io.baseline"
```

Comment:
I don't think we should be checking in a txt file for this. Can this point to a tmp file go be uploaded to GHA as an artifact?

Reply:
I change my mind here. I think checking the file in is fine if the file represents "known" issues. I.e., the PR will fail if there are new issues that don't exist in the baseline file.